tools/pd-ctl/pdctl/command/global_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2024 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package command
+
+import (
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/require"
+)
+
+func TestParseTLSConfig(t *testing.T) {
+	re := require.New(t)
+
+	rootCmd := &cobra.Command{
```

Comment:
I tend to only introduce unit tests for functions like `RequireHTTPSClient`/`requirePDClient`/`parseTLSConfig` in the `global_test.go` file. Introducing cobra commands seems a bit strange here.

Reply:
the input of these function are `cmd *cobra.Command`. If do not introduce it, how do we test it?

Reply:
Or do you mean introducing a new Command?

Reply:
Oops, I didn't notice that.